misaligned/Misaligned.java

```diff
@@ -0,0 +1,31 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+class Misaligned {
+
+    static void printColorMap(Map<Integer, String> colorPair) {
+        colorPair.forEach((k, v) -> System.out.println(k + " " + v));
+    }
+
+    static Map<Integer, String> constructColorPair() {
+        Map<Integer, String> colorPair = new HashMap<>();
+        String[] majorColors = {"White", "Red", "Black", "Yellow", "Violet"};
+        String[] minorColors = {"Blue", "Orange", "Green", "Brown", "Slate"};
+        for (int i = 0; i < majorColors.length; i++) {
+            for (int j = 0; j < minorColors.length; j++) {
+                colorPair.put((i * 5 + j) + 1, " | " + majorColors[i] + " | " + minorColors[j]);
+            }
+        }
+        return colorPair;
+    }
+
+
+    public static void main(String[] args) {
+        Map<Integer, String> colorPair = constructColorPair();
+        printColorMap(colorPair);
+        assert (colorPair.size() == 25);
+        assert (Objects.equals(colorPair.get(14), " | Black | Brown"));

```

Comment:
Good idea to cache the color pair so that it can be verified 👍 
tests need to be cover some more boundary cases like, 0, 1,5,6,10 etc
The input is a function of major color length and minor color length, so boundary values like 1, 5,6,10 should be considered.